_drafts/programatically-configured-bots.md

```diff
@@ -52,9 +52,9 @@ robot.alias = '/'
 # like robot name
 robot.adapter.once 'connected', ->
 
-  # require external hubot scripts
-  robot.loadScriptPackage require('hubot-help')
-  robot.loadScriptPackage require('hubot-shipit')
+  # require external hubot script packages
+  robot.loadPackage 'hubot-help'
+  robot.loadPackage 'hubot-shipit'
```

Comment:
I just ran into that while working on a branch for this actually 😅  For now, I am just doing this, without a function:

```javascript
require('hubot-help')(robot)
```

There is already a `robot.load` function that takes a string path of a script to require and parse the help of. It could be overloaded to also take a function.